### PR TITLE
fix(DB/Creature): Remove incorrect gossip from Roetten Stonehammer

### PR DESCRIPTION
**Pull request by BeefSteaker**
## Changes Proposed:
-  Removes gossip line "My wind riders are trained to fly quickly through the hot barrens air." from Roetten Stonehammer

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13766
- Closes https://github.com/chromiecraft/chromiecraft/issues/4447

## SOURCE:

line belongs to Devrak (3615) https://wowpedia.fandom.com/wiki/Devrak

## Tests Performed:

- teleported to on local
- 

## How to Test the Changes:

1. .go c id 5637

## Known Issues and TODO List:

- [X] ~~Still gives "greetings, $name" gossip despite being zero~~

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
> Still gives "greetings, $name" gossip despite being zero

Seems `npcflag` should be set to 2 instead of 3 for this particular NPC.

**Comment by BeefSteaker** [comment]
> Seems `npcflag` should be set to 2 instead of 3 for this particular NPC according to sniff.

thanks, im still learning where everything is. that fixed it

**Comment by heyitsbench** [comment]
Tested, works as expected.

**Review by Kitzunu — APPROVED** [review]

File: data/sql/updates/pending_db_world/removegossip.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- Remove incorrect gossip from Roetten Stonehammer
+UPDATE `creature_template` SET `gossip_menu_id` = 0 WHERE (`entry` = 5637);
+UPDATE `creature_template` SET `npcflag` = 2 WHERE (`entry` = 5637);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/removegossip.sql:3]:
```suggestion
UPDATE `creature_template` SET `npcflag` = 2, `gossip_menu_id` = 0 WHERE (`entry` = 5637);
```

**Review by Kitzunu — CHANGES_REQUESTED** [review]
Try to always reduce the amount of queries neccessary

**Review by Kitzunu — APPROVED** [review]

**Comment by Kitzunu** [comment]
Thanks for the PR

**Pull request merged**